packages/components/src/focal-point-picker/controls.js

```diff
@@ -42,13 +42,13 @@ export default function FocalPointPickerControls( {
 		<ControlWrapper className="focal-point-picker__controls">
 			<UnitControl
 				label={ __( 'Left' ) }
-				value={ valueX }
+				value={ [ valueX, '%' ].join( '' ) }
 				onChange={ ( next ) => handleChange( next, 'x' ) }
 				dragDirection="e"
 			/>
 			<UnitControl
 				label={ __( 'Top' ) }
-				value={ valueY }
+				value={ [ valueY, '%' ].join( '' ) }
```

Comment:
Is there something I don't know about `join` that makes it better than template literals?

Reply:
I prefer the `join` approach because it ignores `undefined` values:

```js
[ 41, undefined ].join(''); // '41'
`${ 41 }${ undefined }`; // '41undefined'
```

Reply:
Now I get it. Thanks! I'm pretty sure in this case the values cannot be `undefined` but I'm not sweating these `join`s staying put.

Reply:
> I prefer the join approach because it ignores undefined values

Oh, nice! I'm sure I'll borrow 😀